src/test/views/PageLoader.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import PageLoader from '../../views/PageLoader';
+
+describe('<PageLoader />', () => {
+  it('should render the PagLoader', () => {
```

Comment:
#### Nitpick
```suggestion
  it('should render the PageLoader', () => {
```

Reply:
Thanks @aanchirinah , this has been fixed now.